symfony/plugins/orangehrmTimePlugin/Api/TimeReportAPI.php

```diff
@@ -33,6 +34,7 @@ class TimeReportAPI extends ReportAPI
         //activity_detailed -> detailed report of project report (Break down overview of project report)
         'activity_detailed' => ProjectActivityReport::class,
         'employee' => EmployeeReport::class,
+        'attendance' => AttendanceReport::class,
```

Comment:
is this part of time plugin or attendance plugin

Reply:
time?????